### PR TITLE
Scroll to top button added

### PR DESCRIPTION
**Pull request by nehaparmar99**
## Issue that this pull request solves
Added a scroll to top button.
 Closes: #311 

### Brief description of what is fixed or changed
Upon hover the opacity becomes 1, however normally it's set at 50%.
Done under SWOC and DWOC.
## Types of changes

_Put an `x` in the boxes that apply_

- [x] New feature (non-breaking change which adds functionality)

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots
Screenshot (295)
Screenshot (298)

Please attach the screenshots of the changes made in case of change in user interface

## Other information

Any other information that is important to this pull request

### CODE REVIEW

**Comment by nehaparmar99** [comment]
@Kajol-Kumari please review it and let me know if any changes are required

File: frontend/src/app.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { Fragment } from "react";
 import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
 import "semantic-ui-css/semantic.min.css";
 // Pages Import
+import ScrollTop from "./components/scrolltop/scrolltop";
```
Comment (mohit355) [inline frontend/src/app.js:5]: check #311 again.

 folder (component) must be inside the utils folder.

**Review by mohit355 — CHANGES_REQUESTED** [review]

File: frontend/src/app.js (hunk 1 above)
Comment (nehaparmar99) [reply frontend/src/app.js:5]: @mohit355  Done!

**Review by Kajol-Kumari — APPROVED** [review]
LGTM :+1:

**Comment by Kajol-Kumari** [comment]
@nehaparmar99 please mention in the PR description under which program you have made this PR

**Comment by nehaparmar99** [comment]
> @nehaparmar99 please mention in the PR description under which program you have made this PR

I've done it under both SWOC and DWOC.Mentioned it in my description now😅

**Review by mohit355 — COMMENTED** [review]
LGTM

**Pull request merged**